File: to-do app/todo-java/src/main/java/com/renan/todo/dto/TaskDTO.java
package com.renan.todo.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

/**
 * Transfer Object for Task
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class TaskDTO {

    private Integer id;
    private String taskName;
    private String taskDescription;
    private Date deadline;
    private Integer taskStatus;

}

File: to-do app/to-do-go/src/controller/response/task_dto.go
package response

type TaskDTO struct {
	ID              uint64 `json:"id"`
	TaskName        string `json:"taskName"`
	TaskDescription string `json:"taskDescription"`
	Deadline        string `json:"deadline"`
	TaskStatus      uint64 `json:"taskStatus"`
}

type TaskListDTO struct {
	ID              uint64    `json:"id"`
	ListName        string    `json:"listName"`
	ListDescription string    `json:"listDescription"`
	Tasks           []TaskDTO `json:"tasks"`
}
